.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!-- This is a template - do add to or remove from as needed -->
+<!-- Please provide a general summary of your changes in the Title above 🚀 -->
+
```

Comment:
Add a reference to the user story this PR is for

```suggestion
## Context

<!-- Copy the story link from Pivotal Tracker -->
```

Reply:
```suggestion
## Context

[Corresponding Pivotal Tracker story](<!-- Copy the story link from Pivotal Tracker -->)
```